tests/cxx/namespace_macro-direct.h

```diff
@@ -0,0 +1,12 @@
+//===--- namespace_macro-direct.h -----------------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
Oops, sorry I didn't catch this before. We only use `#pragma once` for testing `#pragma once`, please add a real include guard (with the very annoying prefix `INCLUDE_WHAT_YOU_USE_TESTS_CXX_` + filename on symbol form). See other headers for examples.

Reply:
Done